lib/Constants.js

```diff
@@ -227,6 +227,12 @@ Constants.ServiceStatus = {
   ERROR: 'ERROR'
 };
 
+Constants.DeploymentStatus = {
```

Comment:
It seems that these Constant is not used anywhere...

Reply:
https://github.com/Kinvey/kinvey-cli/blob/flex-198-operator-integration/lib/flex/FlexService.js#L34